node-manager/src/nodemanager.rs

```diff
@@ -69,6 +71,9 @@ impl NodeManager {
     pub fn new(password: String, mnemonic: Option<String>) -> NodeManager {
```

Comment:
I think it makes sense for this to return an `anyhow::Result<NodeManager>`, don't think it helps much to keep punting on it at this point

then we turn the expects into `?`

then we can do a try / catch in the browser

Reply:
why `anyhow` ? Why not the mutiny errors?

Reply:
We can do it like this: https://rustwasm.github.io/wasm-bindgen/reference/types/result.html?highlight=result#resultt-e

```
impl Into<JsValue> for MutinyError {
    fn into(self) -> JsValue {
        todo!()
    }
}
```

Reply:
I don't know how to turn all the various errors here into `JsValue` right now. And I don't introduce any additional `expect`'s here so I'll open up a ticket and we can handle it soon outside of this PR.

Reply:
https://github.com/BitcoinDevShop/mutiny-web-poc/issues/38